### ISSUE TITLE
git clone --recurse-submodules not pulling pointfinder_db submod

### ISSUE THREAD

**Issue by ryanjameskennedy** [genomic-medicine-sweden/JASEN#200]
When trying to git clone JASEN the following error occurs:
```
fatal: remote error: upload-pack: not our ref c1d1adf37bf92854a66c6d266d5ae3049b863944
fatal: the remote end hung up unexpectedly
fatal: Fetched in submodule path 'assets/pointfinder_db', but it did not contain c1d1adf37bf92854a66c6d266d5ae3049b863944. Direct fetching of that commit failed.
```

**Comment by samuell** [comment]
Just did a full clone, with `--recurse-submodules` and got another error:

```
Submodule path 'assets/kma': checked out 'beac9fac1771e062326be5e3a2818894d295823b'
fatal: remote error: upload-pack: not our ref c1d1adf37bf92854a66c6d266d5ae3049b863944
fatal: the remote end hung up unexpectedly
fatal: Fetched in submodule path 'assets/pointfinder_db', but it did not contain c1d1adf37bf92854a66c6d266d5ae3049b863944. Direct fetching of that commit failed.
fatal: 
```

**Comment by samuell** [comment]
Have confirmed that this issue is resolved now after the merge of #181
